packages/google_sign_in/google_sign_in/README.md

```diff
@@ -65,6 +65,20 @@ This plugin requires iOS 9.0 or higher.
 <!-- End of the Google Sign-in Section -->
 ```
 
+Instead of adding `GoogleService-Info.plist` to your Xcode project, you can configure your app in
```

Comment:
Let's not remove the "old way" of doing things yet as it will be a breaking change for people who have already been adding keys to `GoogleService-Info.plist`

Reply:
The `GoogleService-Info.plist` continues to be supported. The values for `clientId` and `serverClientId` when provided to `GoogleSignIn` will take precedence over `GoogleService-Info.plist`.